tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/auto_query_fixer/Application.java

```diff
@@ -0,0 +1,14 @@
+package com.google.cloud.bigquery.utils.auto_query_fixer;
+
+public class Application {
+
+  public static void main(String[] args) {
+
+    if (args.length == 0 ) {
+      System.out.println("not enough arguments");
```

Comment:
We should use logger
https://github.com/google/flogger
or java logger https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html

Reply:
https://github.com/GoogleCloudPlatform/bigquery-utils/pull/73/commits/0ebc50fab3773e85f8b30490392fce784f714f88#diff-01227382e8fb4407a726ec6bfcfc3cdcR12